graf2d/asimage/src/libAfterImage/import.c

```diff
@@ -2344,42 +2343,25 @@ tiff2ASImage( const char * path, ASImageImportParams *params )
 static ASImage *
 load_xml2ASImage( ASImageManager *imman, const char *path, unsigned int compression, int width, int height )
 {
-	ASVisual fake_asv ;
-	char *slash, *curr_path = NULL ;
-	char *doc_str = NULL ;
-	ASImage *im = NULL ;
-
-	memset( &fake_asv, 0x00, sizeof(ASVisual) );
-	if( (slash = strrchr( path, '/' )) != NULL )
-		curr_path = mystrndup( path, slash-path );
-
-	if((doc_str = load_file(path)) == NULL )
-		show_error( "unable to load file \"%s\" file is either too big or is not readable.\n", path );
-	else
-	{
-		im = compose_asimage_xml_at_size(&fake_asv, imman, NULL, doc_str, 0, 0, None, curr_path, width, height);
-		free( doc_str );
-	}
-
-	if( curr_path )
-		free( curr_path );
-	return im ;
+   printf("WARNING: Because of some licensing issues load_xml2ASImage is now dummy\n");
```

Comment:
```suggestion
   // Incompatible license
   fprintf(stderr, "ERROR in load_xml2ASImage: XML import is disabled.\n");
```

Reply:
I.e. I think it's nicer:
- to make a dysfunctional call an error,
- to state where the error is,
- to say "disabled" rather than "dummy"
- to not talk about licenses to users but just say it doesn't work
- to print error messages to stderr

Reply:
I applied your suggestions. Thanks.